src/StackAdmin/Azs.Gallery.Admin/readme.md

```diff
@@ -42,23 +42,51 @@ In this directory, run AutoRest:
 > `autorest`
 
 ---
+
 ### AutoRest Configuration
+
 > see https://aka.ms/autorest
 
 ``` yaml
 require:
   - $(this-folder)/../readme.azurestack.md
-  - $(repo)/specification/azsadmin/resource-manager/gallery/readme.md
 
 input-file:
   - $(repo)/specification/azsadmin/resource-manager/gallery/Microsoft.Gallery.Admin/preview/2015-04-01/Gallery.json
   - $(repo)/specification/azsadmin/resource-manager/gallery/Microsoft.Gallery.Admin/preview/2015-04-01/GalleryItem.json
+```
 
-subject-prefix: ''
-module-version: 0.0.1
+### File Renames
 
-### File Renames 
-module-name: Azs.Gallery.Admin 
-csproj: Azs.Gallery.Admin.csproj 
-psd1: Azs.Gallery.Admin.psd1 
+```yaml
+module-name: Azs.Gallery.Admin
+csproj: Azs.Gallery.Admin.csproj
+psd1: Azs.Gallery.Admin.psd1
 psm1: Azs.Gallery.Admin.psm1
+```
+
+### Parameter default values
+
+``` yaml
+directive:
+## variant removal (parameter InputObject) from all New cmdlets -- parameter sets CreateViaIdentity and CreateViaIdentityExpanded
+  - where:
```

Comment:
we have this customization moved to the Top level readme.azurestack.md file as this is applicable for every module

Reply:
I am merging this for now, we can move it later